### PR TITLE
Improve permission middleware: foreach->canany

### PR DESCRIPTION
**Pull request by cesarMtorres**
using canany instead foreach
and other improve
is posible this change???

### CODE REVIEW

**Comment by fredsal** [comment]
> using canany instead foreach

`canany`  Yes, 
but using route's name as permission, Not
 `$permission = $request->route()->getName();`

File: .github/workflows/main.yml [hunk 1]
```diff
@@ -0,0 +1,36 @@
+# This is a basic workflow to help you get started with Actions
+
+name: CI
+
```
Comment (drbyte) [inline .github/workflows/main.yml:4]: This `.github/workflows/main.yml` file you added doesn't seem relevant. Please remove.

**Review by drbyte — CHANGES_REQUESTED** [review]

**Comment by drbyte** [comment]
@cesarMtorres Thanks for the contribution.

Unfortunately `canany()` only works since Laravel 8.  This package is currently still compatible with Laravel 6 and 7.

We may consider your change for a future version which drops support for L6 and L7.

**Comment by cesarMtorres** [comment]
@drbyte thanks for the feedback

**Comment by drbyte** [comment]
@cesarMtorres FYI your `canany` change has been implemented in the upcoming v6

**Pull request closed without merging**